tests/test_aaa_authorization_service.rb

```diff
@@ -22,54 +22,72 @@ class TestAaaAuthorizationService < CiscoTestCase
   def setup
     super
     feature_tacacs
```

Comment:
Your setup needs an initial cleanup as well. We've been using `@@pre_cleanup_needed` as a flag to avoid having setup repeat cleanups after teardown runs, but you still need that very first pre clean. See: https://github.com/cisco/cisco-network-node-utils/blob/develop/tests/test_ace.rb#L22